### PR TITLE
fixed error message capitalization

### PR DESCRIPTION
**Pull request by xuxiawei**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Review by fisherxu — COMMENTED** [review]
It should be start with a lowercase letter for error and uppercase letter for klog :) Because the error will be printed in the klog.

**Comment by xuxiawei** [comment]
> It should be start with a lowercase letter for error and uppercase letter for klog :) Because the error will be printed in the klog.

Modified the uppercase beginning in the error message and the corresponding unit test

File: edge/pkg/devicetwin/process.go [hunk 1]
```diff
@@ -55,7 +55,7 @@ func (dt *DeviceTwin) distributeMsg(m interface{}) error {
 		}
 	}
 	if !classifyMsg(&message) {
-		return errors.New("Not found action")
+		return errors.New("not found action")
```
Comment (gy95) [inline edge/pkg/devicetwin/process.go:58]: /lgtm 
using lower case in error.

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**